pepper-apis/dsm-core/src/test/java/org/broadinstitute/dsm/juniperkits/TestKitUtil.java

```diff
@@ -82,71 +79,72 @@ public class TestKitUtil {
-    private static final UserAdminTestUtil cmiAdminUtil = new UserAdminTestUtil();
-    public static String ddpGroupId;
-    public static String ddpInstanceId;
-    public static String ddpInstanceGroupId;
-    public static String instanceRoleId;
-    public static String ddpInstanceRoleId;
-    static List<String> createdKitIds = new ArrayList<>();
-    private static String kitTypeId;
-    private static String kitDimensionId;
-    private static String kitReturnId;
-    private static String carrierId;
-    private static String ddpKitRequestSettingsId;
-    private static String instanceName;
-    private static String groupName;
-    private static String kitTypeName;
-    private static String studyGuid;
-    private static String collaboratorPrefix;
-    private static String userWithKitShippingAccess;
+    public final UserAdminTestUtil adminUtil = new UserAdminTestUtil();
+    private String kitTypeDisplayName;
+    public Integer ddpGroupId;
+    public Integer ddpInstanceId;
+    public Integer instanceRoleId;
+    public Integer ddpInstanceRoleId;
+    List<String> createdKitIds = new ArrayList<>();
+    public Integer kitTypeId;
+    private Integer kitDimensionId;
+    private Integer kitReturnId;
+    private Integer carrierId;
+    private Integer ddpKitRequestSettingsId;
+    private String instanceName;
+    private String groupName;
+    private String kitTypeName;
+    private String studyGuid;
+    private String collaboratorPrefix;
+    private String userWithKitShippingAccess;
     @Getter
     @Mock
-    static EasyPostUtil mockEasyPostUtil = mock(EasyPostUtil.class);
+    EasyPostUtil mockEasyPostUtil = mock(EasyPostUtil.class);
     @Mock
-    static Address mockEasyPostAddress = mock(Address.class);
+    Address mockEasyPostAddress = mock(Address.class);
     @Mock static  Shipment mockEasyPostShipment = mock(Shipment.class);
     @Mock
-    static Parcel mockEasyPostParcel = mock(Parcel.class);
+    Parcel mockEasyPostParcel = mock(Parcel.class);
     @Mock
-    static PostageLabel mockParticipantLabel = mock(PostageLabel.class);
+    PostageLabel mockParticipantLabel = mock(PostageLabel.class);
     @Mock
-    static Tracker mockShipmentTracker = mock(Tracker.class);
+    Tracker mockShipmentTracker = mock(Tracker.class);
 
     public TestKitUtil(String instanceName, String studyGuid, String collaboratorPrefix, String groupName,
-                                  String kitTypeName) {
+                                  String kitTypeName, String kitTypeDisplayName) {
         this.instanceName = instanceName;
         this.studyGuid = studyGuid;
         this.collaboratorPrefix = collaboratorPrefix;
         this.groupName = groupName;
         this.kitTypeName = kitTypeName;
+        this.kitTypeDisplayName = kitTypeDisplayName;
     }
 
-    public static void deleteInstanceAndSettings() {
+    public void deleteGeneratedData() {
```

Comment:
Renamed because this method is doing more deletes than just instance and settings.